### PR TITLE
Implemented EditMode docs.

### PR DESCRIPTION
**Pull request by pawellysy**

### CODE REVIEW

File: ts/Dashboards/Actions/Resizer.ts [hunk 1]
```diff
@@ -184,7 +187,7 @@ class Resizer {
         const snapHeight = this.options.snap.height || 0;
         const dashboardContainer = this.editMode.board.container;
 
-        // right snap
+        // Right snap
         this.snapXR = createElement(
```
Comment (bre1470) [inline ts/Dashboards/Actions/Resizer.ts:191]: Not in this scope, but why not just name it `snapRight`?

File: ts/Dashboards/Actions/Resizer.ts [hunk 2]
```diff
@@ -544,23 +543,59 @@ interface Resizer {
...
+         * Weather the resizer is enabled or not.
+         */
         enabled: boolean;
-        type: string;
+        /**
+         * Resizer type.
+         */
+        type: 'x'|'y'|'xy';
+        /**
+         * Snap Options.
```
Comment (bre1470) [inline ts/Dashboards/Actions/Resizer.ts:559]: > Options for the snap mechanism.

File: ts/Dashboards/Actions/Resizer.ts [hunk 3]
```diff
@@ -569,7 +604,14 @@ namespace Resizer {
     }
 
     export interface SnapOptions {
```
Comment (bre1470) [inline ts/Dashboards/Actions/Resizer.ts:606]: > Options for the snap mechanism.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍 

Just some suggestions...

**Review by sebastianbochan — APPROVED** [review]
Please merge with `dashboards/main` to ensure that Cypress tests are passed properly.

**Pull request merged**